src/components/common/header/PageHeader.tsx

```diff
@@ -159,7 +160,7 @@ function PageHeader({
                     </span>
                     {additionalHeaderInfo && additionalHeaderInfo()}
                     {isBreadcrumbs && breadCrumbs()}
-                    {isTippyShown && (
+                    {isTippyShown && headerName != "Bulk Edit" && (
```

Comment:
create constant for bulk edit and use it here and at the bottom

Reply:
done